### ISSUE TITLE
Scan not working with dremio type

### ISSUE THREAD

**Issue by carlosnizolli** [sodadata/soda-core#1693]
When running with docker the error below occurs
![image](https://user-images.githubusercontent.com/55762776/202299819-27d52d1d-d15f-433b-b88f-536f8874b403.png)

When I try to run local I get this error:
![image](https://user-images.githubusercontent.com/55762776/202300361-1eb1f98e-0466-4b58-a511-5ed57c72b9ee.png)

**Comment by jmarien** [comment]
SODA-1309

**Comment by vijaykiran** [comment]
@carlosnizolli  Did you already install https://docs.dremio.com/software/drivers/arrow-flight-sql-odbc-driver/ ?

**Comment by vijaykiran** [comment]
Oh - wait - you are running it with docker image .. ?

**Comment by carlosnizolli** [comment]
I tried both ways. Locally I didn't install the driver because it is not in the documentation as required. With docker the first error occurs.

**Comment by carlosnizolli** [comment]
For us, the priority would be to work on docker to run in the cloud.

**Comment by vijaykiran** [comment]
@carlosnizolli  I'll release a  new version of docker image/agent today

**Comment by vijaykiran** [comment]
@carlosnizolli  Did you install 0.8.13 version of agent?

**Comment by carlosnizolli** [comment]
@vijaykiran I'm following the instructions in the documentation https://docs.soda.io/soda-core/installation.html#use-docker-to-run-soda-core

**Comment by vijaykiran** [comment]
@carlosnizolli  I have only updated the soda-agent, soda-core docker image is something we provided as a quick workaround for the soda-core scientific and not all packages were added to that. 

For production deployments - please use the Agent via Helm, I'll try to update the Soda Core docker in a few hours.

**Comment by carlosnizolli** [comment]
@vijaykiran We do not use Soda agent. To run with soda-core-dremio installed is it necessary to install more requirements that are not in the documentation?

**Comment by vijaykiran** [comment]
Yes, you need to install https://www.dremio.com/drivers/odbc/ locally for the dremio connector to work.

**Comment by carlosnizolli** [comment]
This I installed. And I added the driver in the configuration file

**Comment by vijaykiran** [comment]
Notes : soda-core-dremio only supports 22+ because we use PyArrow ODBC SQL driver by Dremio, which only supports Dremio 22+
